src/appShell/App/Container.tsx

```diff
@@ -137,6 +137,27 @@ export default class Container extends React.Component<IContainerProps, {}> {
                         </div>
                     </Else>
                 </If>
+                <If condition={this.appStore.unknownOrUnauthorizedStudyIds}>
```

Comment:
we shouldn't need to do this in the container.   the study view page itself can return an <ErrorScreen> component under this condition.